src/apiCalls.js

```diff
@@ -5,8 +5,12 @@ const apiCalls = {
       if (response.ok) {
         return response.json()
       } else {
-        console.log(response)
-        throw `${response.status} ${response.statusText}`
+        console.log('responseStatus', response.status)
+        if(response.status < 500) {
+          throw `We're sorry, something went wrong. Either the page doesn't exist, or could not be found.`
```

Comment:
Reafactored the error handling to distinguish between 500 type erros and all others. Could probably come up with better error messages in the future.